src/readme.html

```diff
@@ -3,57 +3,59 @@
 <head>
 	<meta name="viewport" content="width=device-width" />
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-	<title>WordPress &#8250; ReadMe</title>
-	<link rel="stylesheet" href="wp-admin/css/install.css?ver=20100228" type="text/css" />
+	<title>ClassicPress &#8250; ReadMe</title>
+	<link rel="stylesheet" href="wp-admin/css/install.css?ver=20181218" type="text/css" />
 </head>
 <body>
 <h1 id="logo">
-	<a href="https://wordpress.org/"><img alt="WordPress" src="wp-admin/images/wordpress-logo.png" /></a>
+	<a href="https://www.classicpress.net/">ClassicPress</a>
 </h1>
-<p style="text-align: center">Semantic Personal Publishing Platform</p>
+<p style="text-align: center">The business-focused CMS.<br>Powerful. Versatile. Predictable.</p>
 
-<h2>First Things First</h2>
-<p>Welcome. WordPress is a very special project to me. Every developer and contributor adds something unique to the mix, and together we create something beautiful that I&#8217;m proud to be a part of. Thousands of hours have gone into WordPress, and we&#8217;re dedicated to making it better every day. Thank you for making it part of your world.</p>
-<p style="text-align: right">&#8212; Matt Mullenweg</p>
+<h2>A word from our Founder</h2>
+<p>[we need a paragraph intro text from Scott for this]</p>
```

Comment:
@senlin whatever we get done in this PR is probably going to ship in v1, so it should not be "we need a paragraph from Scott for this".  Can you work with @scottybo to get something we can drop in here, or should we remove this section?

Reply:
@nylen when I commited this more than a month ago it was still a valid "we need a paragraph from Scott for this".
I will see what I can get from him and update this ticket accordingly. Do we know when v1 ships?

Reply:
v1 can ship when all the things in https://github.com/ClassicPress/ClassicPress/milestone/5 and https://github.com/ClassicPress/ClassicPress/milestone/3 are done.  This PR is on the list, thanks for your work on it so far.

We don't have an ETA right now  Progress has slowed down due to the holidays and me having to focus more on client work.